builders/relayedTxV3Builder.go

```diff
@@ -0,0 +1,79 @@
+package builders
+
+import (
+	"github.com/multiversx/mx-chain-core-go/data/transaction"
+	"github.com/multiversx/mx-sdk-go/data"
+)
+
+type relayedTxV3Builder struct {
+	innerTransaction *transaction.FrontendTransaction
+	relayerAccount   *data.Account
+	networkConfig    *data.NetworkConfig
+}
+
+// NewRelayedTxV3Builder creates a new relayed transaction v2 builder
+func NewRelayedTxV3Builder() *relayedTxV3Builder {
+	return &relayedTxV3Builder{
+		innerTransaction: nil,
+		relayerAccount:   nil,
+		networkConfig:    nil,
+	}
+}
+
+// SetInnerTransaction sets the inner transaction to be relayed
+func (rtb *relayedTxV3Builder) SetInnerTransaction(tx *transaction.FrontendTransaction) *relayedTxV3Builder {
+	rtb.innerTransaction = tx
```

Comment:
you could return an error here if the `relayerAccount` field is empty

Reply:
added this check on Build method in order to keep the same structure as other builders(no checks on SetInnerTransaction, but all on Build)